images/base/Dockerfile

```diff
@@ -142,6 +143,7 @@ RUN echo "Installing containerd ..." \
     && sha256sum --ignore-missing -c /tmp/runc.sha256 \
     && mv /tmp/runc.${TARGETARCH} /usr/local/sbin/runc \
     && chmod 755 /usr/local/sbin/runc \
+    && ctr oci spec > /etc/containerd/cri-base.json \
```

Comment:
do we need to generate the base spec?
is this going to have specific details that may not be portable?
I'm not too  much into the OCI spec details, but it will be ideal if we can just create a generic file with the bind mounts (we already know the paths) and drop it in the repo in `base/files/etc/containerd/cri-base.json`

Reply:
In case of chaning containerd/runc version genereting it ensure we are using the expected defaults/format, but this is just a guess a being super safe, maybe it's not needed at all, also the value is calculated at entrypoint so we need to modify it there.

Reply:
@dims can you help us here? ^^^

Reply:
The main problem we have right now is CI, looks like the trick with /sys/class/dmi/id/product_uuid breaks CI, I have run it locally and something is fishy there but was not able to see anything relevant at containerd journalctl.

It feels like latency accessing containers get increased, maybe now that they share a bind mount introduce this problem ? 

product_name or /sys/devices/virtual/dmi/id/product_uuid works fine.

Reply:
ACK. will find time today 🤞🏾

Reply:
do we still need to generate this on the fly with the new approach?

Reply:
Not really needed, we can put a modified cri-base.json with the hook, although the "jq" is now also used on the hook script to parse the OCI container json and get the bundle, what's your concern with oci spec generation ?

Reply:
less dependencies, less moving parts, less complexity ...

Reply:
Do you want to also remove jq and try to parse with bash or use the "jq" container so we don't install it on the image ?

Reply:
@BenTheElder is the master of that area, let's see what he things ^^

Reply:
I am adding a pair of commits with the proposed changes so we can easily revert if we are not ok with that.